src/test/java/api/loans/agetolost/ScheduledAgeToLostFeeChargingApiTest.java

```diff
@@ -56,6 +64,12 @@ public void shouldChargeItemFee() {
     assertThat(result.getLoan().getJson(), isLostItemHasBeenBilled());
     assertThat(result.getLoan(), hasLostItemFee(isOpen(expectedSetCost)));
     assertThat(result.getLoan(), hasLostItemFeeCreatedBySystemAction());
+
+    assertThat(result.getLoan(), hasLoanHistory(hasItems(
+      hasJsonPath("loan.action", ""),
+      hasJsonPath("loan.action", "itemAgedToLost"),
+      hasJsonPath("loan.action", "checkedout")
+    )));
```

Comment:
Does this match the items in order, or could the items appear in any order and still pass this matcher?

Reply:
It does not match order.

Reply:
Does that mean that this will succeed if there is any entry in the history with no action?

Reply:
yes, it is true.

Reply:
Could this be adapted to have a stricter check where the order matters, specifically, it is the last one being blank that matters?

Reply:
Done